core/src/client/importer.rs

```diff
@@ -371,26 +371,52 @@ impl Importer {
         imported.len()
     }
 
-    pub fn import_bootstrap_block<'a>(&'a self, block: &'a Block, client: &Client, _importer_lock: &MutexGuard<()>) {
+    pub fn import_trusted_header<'a>(&'a self, header: &'a Header, client: &Client, _importer_lock: &MutexGuard<()>) {
```

Comment:
Why it takes unused `_importer_lock`?

Reply:
I don't know why. I just used the same interface as `import_headers`.